Content.Server/Mail/Components/MailTeleporterComponent.cs

```diff
@@ -71,13 +71,13 @@ public sealed partial class MailTeleporterComponent : Component
     ///     What's the bonus for delivering a fragile package intact?
     /// </summary>
     [DataField]
-    public int FragileBonus = 100;
+    public int FragileBonus = 1000;
```

Comment:
This is iffy. There's only so many ways to damage fragile mail that wouldn't also damage normal mail. Typically, unless you throw mail at people, it won't get damaged.